src/index.spec.ts

```diff
@@ -84,15 +80,30 @@ it('should throw if the validation fails', () => {
   );
 });
 
-it('should describe all registered variables', () => {
+it('should throw if a post validation rule fails', () => {
+  const envaridator = new Envaridator();
+  envaridator.registerPostValidation(
+    'THROW_RULE',
+    () => {
+      throw new Error('This rule should throw');
+    },
+    'This rule will throw when the envaridator validate is run',
+  );
+
+  expect(() => envaridator.validate()).toThrow(
```

Comment:
PostValidation shouldn't say "the following environment variables are invalid"

Reply:
Fixed.

Reply:
but the test isn't updated?

Reply:
oh i see it, never mind.